### ISSUE TITLE
According to Firebase Documentations setPersistenceEnabled is to be called only once

### ISSUE THREAD

**Issue by shortstopyomin** [neiyo1987/QandA#1]
http://stackoverflow.com/questions/37753991/com-google-firebase-database-databaseexception-calls-to-setpersistenceenabled

According to Firebase Documentations setPersistenceEnabled is to be called only once (before any other instances of FirebaseDatabase are made)

**Comment by jp-wang** [comment]
Yep... That's why i added a protection around the setPersistenceEnabled to make sure it was only initialized once.
